packages/fether-electron/src/main/app/cli/index.js

```diff
@@ -31,24 +32,6 @@ cli
     '--no-run-parity',
     `${productName} will not attempt to run the locally installed parity.`
   )
-  .option(
-    '--ws-port <port>',
-    `Specify the port portion of the WebSockets server ${productName} will connect to. (default: ${DEFAULT_WS_PORT})`,
-    DEFAULT_WS_PORT
-  )
-
-  .parse(
-    process.argv
-      // We want to ignore some flags and not pass them down to Parity:
-      // --inspect: `electron-webpack dev` runs Electron with the `--inspect` flag for HMR
-      // -psn_*: https://github.com/paritytech/fether/issues/188
-      // --ws-interface: we don't pass down this flag, because fether only allows 127.0.0.1 as WS interface
-      .filter(
-        arg =>
-          !arg.startsWith('--inspect') &&
-          !arg.startsWith('-psn_') &&
-          !arg.startsWith('--ws-interface')
-      )
-  );
+  .parse(process.argv);
```

Comment:
My hunch is that Electron or MacOs would be pass down flags to Electron, which are unexpected on our side and would make the program exit with error code != 0 (because we don't allow unknown arguments anymore). I think I would keep the filtering of `process.argv` ?

```js
parse(
    process.argv
      // We want to ignore some flags and not pass them down to Parity:
      // --inspect: `electron-webpack dev` runs Electron with the `--inspect` flag for HMR
      // -psn_*: https://github.com/paritytech/fether/issues/188
      // --ws-interface: we don't pass down this flag, because fether only allows 127.0.0.1 as WS interface
      .filter(
        arg =>
          !arg.startsWith('--inspect') &&
          !arg.startsWith('-psn_') &&
          !arg.startsWith('--ws-interface')
      )
```

Reply:
It seems that `--inspect` is not passed by `electron-webpack` anymore, so we can remove that one. `--ws-interface` has been removed.

You're probably right, i'll keep `arg => !arg.startsWith('-psn_')` just to be sure